src/components/data/toolbar/Toolbar.js

```diff
@@ -61,6 +68,10 @@ function DataToolbar(props) {
     const onCreateFolderClicked = () => setCreateFolderDlgOpen(true);
     const selectedResources = getSelectedResources();
     const canShare = isOwner(selectedResources);
+    const theme = useTheme();
+    const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
```

Comment:
I always use `xs` down for checking mobile screen sizes. I tested this PR locally and it seems to render stuff correctly but I would like to standardize on breakpoints to provide a uniform UX. Does using `xs` cause issues in iPads like screen sizes ?

Reply:
I could rename this to `isSmall` if that's clearer - I agree that `xs` is the real mobile size so the variable name is probably confusing.  

This is set to `sm` because the buttons in the toolbar were already set to hide with `<Hidden smDown...`, so the `isMobile` is used to match that for the dot menu. If I switch to `xs` then things get cramped before they hide into the dot menu:
<img width="610" alt="image" src="https://user-images.githubusercontent.com/8909156/95504585-d4b45d80-0961-11eb-831c-9cc11d5cde30.png">

Reply:
Renaming the variable sounds good to me 👍. One last thing - Setting the toolbar button size to `small` might also help with real estate.

Reply:
Looks like `small` helps a little bit, but still is cramped within the `sm` range:
<img width="599" alt="image" src="https://user-images.githubusercontent.com/8909156/95511629-940e1180-096c-11eb-99f2-9e424b03ca6a.png">

Reply:
Thanks for giving it shot. So let's settle for `sm` break point. But can we still keep the Button size set `small`. I think it definitely will help breadcrumbs (large paths) getting little bit more space.